doc/api/modules.md

```diff
@@ -841,6 +841,14 @@ added: v9.3.0
 A list of  the names of all modules provided by Node.js. Can be used to verify
 if a module is maintained by a third-party module or not.
 
+Note that `module` in this context isn't the same object that's
+available by default inside file modules. To access it you need to
```

Comment:
Instead of saying "available by default inside file modules" maybe say "provided by the module wrapper" and you can link to the module wrapper section.

Reply:
Fixed